utils/llm.py

```diff
@@ -64,7 +63,7 @@ def token_cost(model: str) -> tuple[float, float]:
     return cost_info
 
 
-def response_cost(response: openai.ChatCompletion) -> float:
+def response_cost(response: ModelResponse) -> float:
```

Comment:
`litellm` has [several functions](https://docs.litellm.ai/docs/completion/token_usage) that could replace ours, like `completion_cost`.